posthog/migrations/0436_alter_plugin_organization.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.11 on 2024-07-09 16:15
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0435_alter_plugin_url"),
```

Comment:
Make these a single migration. The more migrations we have the longer local dev setup takes, so less is better, especially if they touch the same table.

Reply:
Done.